nr1.json

```diff
@@ -1,6 +1,6 @@
 {
   "schemaType": "NERDPACK",
-  "id": "4a13b4e8-3129-467b-a099-ba5f503fe47f",
+  "id": "230df80a-007d-45ea-80aa-3b92d0b7ad94",
```

Comment:
Is this ID meant to be changed? @jpvajda 

I was just following the README

```bash
nr1 nerdpack:clone -r https://github.com/newrelic/nr1-groundskeeper.git
cd nr1-groundskeeper
nr1 nerdpack:serve
```

Reply:
Right, those directions are good for if someone is looking to create a new repo based on this, not if making changes in this repo itself. So the ID should not be changed in this repo.

Reply:
Got it, thanks Tim!